docs/table/access_table.rst

```diff
@@ -672,9 +670,7 @@ bytestring data in astropy Table and Column
 
 Taking this further, there is an *advantage* to using Python 3 because it is
```

Comment:
Just remove the whole paragraph?

Reply:
During the coordination meeting, it was said that it is okay (and perhaps even advantageous) to keep reference of how awesome Python 3 is to encourage Python 2 users to switch over. So, let's keep this for now.

Reply:
But it is not clear any more that a comparison with python 2 is being made...

Reply:
Hmm. What do you think @bsipocz and @eteq ?

Reply:
I vote and voted for remove any mention of python2, just as it never existed. We well over the time when needed to convince the users to switch. Now they either with python3 or need to stuck on the LTS release and thus see a different version of the docs.